K2fov/data/k2-campaign-parameters.json

```diff
@@ -206,5 +206,15 @@
     "start": "2018-08-06",
     "stop": "2018-10-10",
     "comments": "Forward-facing; Trappist-1, Neptune."
+  },
+
+  "c20": {
+   "campaign": 20,
+   "ra": 71.2012890,
+   "dec": 23.1361165,
+   "roll": 7.9176424,
+   "start": "2018-10-11",
```

Comment:
Let's make this `"start": "2018-10-15"` for now.

Reply:
Sure

Reply:
Actually: this looks like the wrong field to me.  Let's make sure we use the correct ra/dec/roll parameters for option 1 here.